boards/limifrog-v1/include/periph_conf.h

```diff
@@ -0,0 +1,197 @@
+#define I2C_0_CLKDIS()      (RCC->APB1ENR &= ~(RCC_APB1ENR_I2C1EN))
+#define I2C_0_EVT_IRQ       I2C1_EV_IRQn
+#define I2C_0_EVT_ISR       isr_i2c1_ev
+#define I2C_0_ERR_IRQ       I2C1_ER_IRQn
+#define I2C_0_ERR_ISR       isr_i2c1_er
+/* I2C 0 pin configuration */
+#define I2C_0_PORT_CLKEN()  (RCC->AHBENR |= RCC_AHBENR_GPIOBEN)
+#define I2C_0_PORT          GPIOB
+#define I2C_0_SCL_PIN       8
+#define I2C_0_SCL_AF        4
+#define I2C_0_SDA_PIN       9
+#define I2C_0_SDA_AF        4
+/** @} */
+
+/**
+ * @name GPIO configuration
+ * @{
+ */
+#define GPIO_0_EN           1
+#define GPIO_1_EN           1
+#define GPIO_2_EN           1
+#define GPIO_3_EN           1
+#define GPIO_IRQ_PRIO       1
+
+/* IRQ config */
+#define GPIO_IRQ_0          GPIO_3
+#define GPIO_IRQ_1          0                      /* not configured */
+#define GPIO_IRQ_2          0                      /* not configured */
+#define GPIO_IRQ_3          0                      /* not configured */
+#define GPIO_IRQ_4          0                      /* not configured */
+#define GPIO_IRQ_5          0                      /* not configured */
+#define GPIO_IRQ_6          0                      /* not configured */
+#define GPIO_IRQ_7          0                      /* not configured */
+#define GPIO_IRQ_8          GPIO_1
+#define GPIO_IRQ_9          0                      /* not configured */
+#define GPIO_IRQ_10         0                      /* not configured */
+#define GPIO_IRQ_11         0                      /* not configured */
+#define GPIO_IRQ_12         GPIO_2
+#define GPIO_IRQ_13         0                      /* not configured */
+#define GPIO_IRQ_14         0                      /* not configured */
+#define GPIO_IRQ_15         GPIO_0
+
+
+/* GPIO channel 0 config */
+#define GPIO_0_PORT         GPIOA                   /* user button 1 */
+#define GPIO_0_PIN          15
+#define GPIO_0_CLKEN()      (RCC->AHBENR |= RCC_AHBENR_GPIOAEN)
+#define GPIO_0_EXTI_CFG()   (SYSCFG->EXTICR[3] |= SYSCFG_EXTICR4_EXTI15_PA)
+#define GPIO_0_IRQ          EXTI15_10_IRQn
+/* GPIO channel 1 config */
+#define GPIO_1_PORT         GPIOC                   /* user button 2 */
+#define GPIO_1_PIN          8
+#define GPIO_1_CLKEN()      (RCC->AHBENR |= RCC_AHBENR_GPIOCEN)
+#define GPIO_1_EXTI_CFG()   (SYSCFG->EXTICR[2] |= SYSCFG_EXTICR3_EXTI8_PC)
+#define GPIO_1_IRQ          EXTI9_5_IRQn
+/* GPIO channel 2 config */
+#define GPIO_2_PORT         GPIOC
+#define GPIO_2_PIN          12
+#define GPIO_2_CLKEN()      (RCC->AHBENR |= RCC_AHBENR_GPIOCEN)
+#define GPIO_2_EXTI_CFG()   (SYSCFG->EXTICR[3] |= SYSCFG_EXTICR4_EXTI12_PC)
+#define GPIO_2_IRQ          EXTI15_10_IRQn
+/* GPIO channel 3 config */
+#define GPIO_3_PORT         GPIOB
+#define GPIO_3_PIN          0
+#define GPIO_3_CLKEN()      (RCC->AHBENR |= RCC_AHBENR_GPIOAEN)
+#define GPIO_3_EXTI_CFG()   (SYSCFG->EXTICR[0] |= SYSCFG_EXTICR1_EXTI0_PB)
+#define GPIO_3_IRQ          EXTI0_IRQn
+
```

Comment:
In  #3095 we slightly changed the gpio interface and implementation "style". As the stml1 and the nucleo-l1 board are still in the old fashion, I think it's ok to go this way. In a second step (PR) I'd recommend to adapt both ;-)